sgpc3/sgpc3.h

```diff
@@ -35,41 +35,280 @@
 #include "sensirion_common.h"
 #include "sensirion_i2c.h"
 
+#define SGPC3_ERR_INSUFFICIENT_FEATURE_SET (-11)
```

Comment:
Can we rename "INSUFFICIENT" to "UNSUPPORTED"? There's nothing wrong with the hardware, we just don't want to clutter this driver with hardware that most people will be unlikely to possess.

Reply:
Dito